features.fea

```diff
@@ -133,3 +133,12 @@ feature ccmp {
 #        pos base [a n] <anchor 307 573> mark @mark_top;
 #    } mark1;
 #} mark;
+
+@roundlc = [ @b @d @h @o @p eth thorn ];
+
+feature calt {

```

Comment:
Since this feature is enabled by default, this should be `hist` (historical forms), which is not default (most of the time, I _think_ it's possible to tell Fontconfig to request to enable it/any OT feature in every font on the system), as otherwise people's documents will change.

Reply:
Oh, I should inform you, contrary to popular belief, any OpenType statement works in any feature tag; some people think that kerning only works in `kern`, contextuals in `calt` or ligatures in `liga`; this is actually not the case. :)

Reply:
Heh... yeah, I knew that, but admittedly it's comparatively new knowledge foꝛ me.  I had been taking Fontfoꝛge's woꝛd foꝛ what goes where until I found that you could type in a feature tag against its advice and things still woꝛked.  But c'mon, who *wouldn't* want to have their r's replaced with ꝛ's? ☺
Changed to `hist`.